src/libMessage/Messenger.cpp

```diff
@@ -5396,6 +5476,140 @@ bool Messenger::GetLookupSetTxnsFromLookup(
   return true;
 }
 
+bool Messenger::SetLookupGetDirectoryBlocksFromSeed(vector<unsigned char>& dst,
+                                                    const unsigned int offset,
+                                                    const uint32_t portno,
+                                                    const uint64_t& index_num) {
```

Comment:
portno and index_num naming style not consistent, suggest change to portNo and indexNum